src/app/seamly2d/xml/vpattern.cpp

```diff
@@ -3964,7 +3967,7 @@ QRectF VPattern::ActiveDrawBoundingRect() const
                 case Tool::PointOfIntersectionCurves:
                 case Tool::PointFromCircleAndTangent:
                 case Tool::PointFromArcAndTangent:
-                    rec = ToolBoundingRect<VToolSinglePoint>(rec, tool.getId());
+                    rect = ToolBoundingRect<VToolSinglePoint>(rect, tool.getId());
```

Comment:
good catch on this!

Reply:
Well, that's why spelling and good choice of variable names matter. It's clear now this is a rectangle and not a record.